src/Response.php

```diff
@@ -5,19 +5,28 @@
 class Response
 {
     private $data;
+
+    private $dataObject;
+
     private $errors;
 
-    public function __construct(array $data, array $errors = [])
+    public function __construct(array $data, array $dataObject, array $errors = [])
```

Comment:
Can we make it backward compatible?

Reply:
I think this is an internal interface, so we don't need to do it backwards compatible.

Reply:
Jose is true.